### DISCUSSION TITLE
Help with setting up a repository stats action

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#835] [Q&A] [answered]
My current goal is to setup an action which would generate and display statistics related to a repository on that repository's readme file.

I'm a bit confused on what I need to change in the GitHub Action, in order to actually display stuff properly.

For context: The repository that I want to apply this towards is [Readme-Workflows/recent-activity](https://github.com/Readme-Workflows/recent-activity) and I would like to display recent (and total) stars the repository has and contributors towards it.

How should the action look like to achieve this?

Also, maybe add a guide to explain repository-based metrics?

**Comment by Andre601** [comment]
I've made a PR that has the planned changes and I would appreciate if you take a look and tell me, if it is good or needs changes:
https://github.com/Readme-Workflows/recent-activity/pull/152

**Reply by lowlighter** [reply]
I added some comments on it 👍

**Comment by lowlighter** [comment, marked as answer]
Yeah the repo feature is not explained very well in other templates other than the repository one.

But basically it's the same as for user, except that you need to provide `repo` option with repository name and set `user` to owner.
Maybe it'd be a good feature to make it so `repo` option is able to parse a repo handle directly
